andela/mrm_api
# #164266032 Add structure folder to backend api

#### What does this PR do?
- Add structure folder to backend API
#### Description of Task to be completed?
- add structure folder to the backend API
- create a model to represent office structure levels using Tree
#### How should this be manually tested?
- git pull branch `ft-office-structure-tree-model-164266032` and check it out
- run `make build`
#### Any background context you want to provide?
- Here are the few resources
    - `https://sqlalchemymptt.readthedocs.io/_modules/sqlalchemy_mptt/mixins.html#BaseNestedSets`
    - `https://github.com/uralbash/sqlalchemy_mptt`
    - `https://sqlalchemy-mptt.readthedocs.io/tut_flask.html`
#### What are the relevant pivotal tracker stories?
[#164266032](https://www.pivotaltracker.com/story/show/164266032)

#### context*
NB: I haven't run migrations yet because the branch am branching off from currently has errors In the Alembic file
